examples/nidigital_spi/measurement.py

```diff
@@ -104,10 +104,9 @@ def measure(
                 str(_resolve_relative_path(service_directory, pattern_file_path)),
             )
 
-        selected_sites.apply_levels_and_timing(
-            str(_resolve_relative_path(service_directory, levels_file_path)),
-            str(_resolve_relative_path(service_directory, timing_file_path)),
-        )
+        levels_file_name = _get_file_name_from_path(levels_file_path)
+        timing_file_name = _get_file_name_from_path(timing_file_path)
```

Comment:
I guess we don't need a helper function for this after all.

```suggestion
        levels_file_name = pathlib.Path(levels_file_path).stem
        timing_file_name = pathlib.Path(timing_file_path).stem
```

Reply:
Removed the helper